### ISSUE TITLE
when trying to use validateAll get error

### ISSUE THREAD

**Issue by closet6** [poppinss/indicative#6]
had this line of code and it worked except the messages.

```
indicative.validate(rules, request.body, valMessages)
```

when i switched it to this:

```
indicative.validateAll(rules, request.body, valMessages)
```

i got this error:

```
Unhandled rejection TypeError: Cannot read property 'toString' of undefined
at Validator.ExtendedRules.validations.min...
extendedRules.js:490:34)
```

version: "indicative": "^1.0.8",

**Comment by thetutlage** [comment]
Show me your rules ?

**Comment by closet6** [comment]
sure

```
var schema = {
        title: 'required|min:10|max:200',
        description: 'required|min:10|max:2000'
    };
```

**Comment by thetutlage** [comment]
Fixed this issue as of `1.0.9`

**Comment by closet6** [comment]
Updated it, and worked in my tests.  Thanks.
